cfpb/cfgov-refresh
# Ask landing pages

Adds Spanish and English Ask landing page content

## Additions

- Spanish and English Ask landing page content
- English landing page hero is editable in Wagtail
- Equal-height columns
- Partially underlined links

## Changes

- Home page cards equal height at desktop sizes

## Testing

- Check out branch
- Set up Ask if necessary
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- navigate to [ask-cfpb](http://localhost:8000/ask-cfpb/) and [obtener-respuestas](http://localhost:8000/obtener-respuestas/) and verify that they work as expected
- navigate to 'Ask CFPB' English landing page in Wagtail and test adding a hero

## Screenshots

![ask cfpb consumer financial protection bureau 1](https://cloud.githubusercontent.com/assets/778171/25492400/c582a946-2b27-11e7-8f64-26033b5cc858.png)
![ask cfpb consumer financial protection bureau 2](https://cloud.githubusercontent.com/assets/778171/25492412/cc4429da-2b27-11e7-9dba-07fb2dcbbe4d.png)
![obtener respuestas oficina para la proteccion financiera del consumidor](https://cloud.githubusercontent.com/assets/778171/25492414/cf71d508-2b27-11e7-81cf-90f6181a645a.png)
<img width="1251" alt="screen shot 2017-04-27 at 8 35 19 am" src="https://cloud.githubusercontent.com/assets/778171/25492418/d4a1f26a-2b27-11e7-9e8e-0968cd5f9869.png">



## Notes
- In this version of the English landing page I build most of the page content (cards, mobile nav, tags) inside the landing page template, but I'm working on a second pass that uses Wagtail components for some or all of the pieces.
- In the existing Ask landing page, we use a jQuery polyfill to create equal height columns in browsers that do not support flexbox (IE8, IE9, etc). I think the Ask landing page layout gracefully degrades reasonably well to inline-block cards with different heights, so I haven't ported that functionality here -- although I can if there's preference for it. 

## Checklist
